.github/workflows/electron.yml

```diff
@@ -29,14 +29,34 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
+      - name: Download certificate file
+        run: |
+          echo $APPLE_DEVELOPER_ID_FILE | base64 --decode >> developer-id.privatekey.p12
+        if: matrix.os == 'macos-latest'
+        shell: bash
+        env:
+          APPLE_DEVELOPER_ID_FILE: ${{secrets.APPLE_DEVELOPER_ID_FILE}}
+
       - name: Build for production
         run: pnpm build:electron
 
-      - name: Building the Electron app
+      - name: Building the Electron app for Mac
+        run: npx electron-builder
+        if: matrix.os == 'macos-latest'
+        env:
+          CSC_LINK: developer-id.privatekey.p12
+          CSC_KEY_PASSWORD: ${{secrets.APPLE_DEVELOPER_ID_PASSWORD}}
+          APPLE_ID: ${{secrets.APPLE_ID}}
+          APPLE_ID_PASSWORD: ${{secrets.APPLE_ID_PASSWORD}}
+          APPLE_TEAM_ID: KJ6LCYQ3A8
+
+      - name: Building the Electron app for Windows
         run: npx electron-builder
+        if: matrix.os == 'windows-latest'
```

Comment:
Please try adding back Ubuntu and change this condition to run for both.. it does doesn't really cost us anything to have it, does it..?

Reply:
Doesn't cost anything. But do we have anybody that can verify if it's working on Linux?
If not we can't publish it anyway.

Reply:
@Ghostbird could install the version I created earlier, so I don't know if there's anything additionally needed. Seems to be just fine using the same commands as for windows (which didn't need any changes, right?)